zq2/docs/nodes/nodefaq.md

```diff
@@ -0,0 +1,41 @@
+---
+id: nodes/nodefaq
+title: Frequently Asked Questions
+keywords:
+  - FAQ
+  - Questions
+  - NODE 
+description: Frequently asked questions
+---
+
+# Frequently Asked Questions
+
+## [Validator node FAQ](#validator-faq)
+## How do I sync my validator node? Should I use genesis sync or checkpoint sync ?
+
+Currently, only genesis mode is supported for syncing validator nodes. We are actively working
+on a fix for this issue. Issue link: [issue link]. Once the fix is live, you can use both genesis
+sync and checkpoint sync.
+
+## [API node FAQ](#api-node-faq)
+## How do i sync my API node? Should i use genesis sync or checkpoint sync ?
+For API node, both genesis sync and checkpoint sync option are supported.
+
+## My API node s stuck at a specific block and is not progressing. What should i do ?
+Try restarting the docker container, and the node should resume syncing from the last block.
```

Comment:
typos and removal of API: My node `is` stuck at a specific block and is not progressing. What should `I` do ?

Reply:
Fixed

Reply:
can you fix the other one too pls: What should `I` do ?